bump-tag

```diff
@@ -1,66 +1,268 @@
 #!/bin/bash
 
-set -e
-
-echo "Fetching any updates from server:"
-git pull
+echo "Fetching updates from $(git remote get-url origin) ..."
 echo ""
+git pull --verify-signatures
```

Comment:
shellchk suggests:

```
if ! git pull --verify-signatures; then
    echo "WARNING: git pull did not exit successfully."
    echo ""
    echo "EXITING the script. In order to tag your changes,"
    echo "investigate and then run bump-tag again."
    exit 1
fi
```

Reply:
I agree and have changed all instances of this pattern.